src/core/BufferGeometry.js

```diff
@@ -334,6 +334,23 @@ Object.assign( BufferGeometry.prototype, EventDispatcher.prototype, {
 
 	},
 
+	setFromPoints: function ( points ) {
+
+		var position = [];
+
+		for ( var i = 0, l = points.length; i < l; i ++ ) {
+
+			var point = points[ i ];
+			position.push( point.x, point.y, 0 );
```

Comment:
How come we are not checking `point.z` here?

Reply:
Whoops! That's a bug. Fixed with the latest commit.